pkg/stripe/analytics_telemetry.go

```diff
@@ -231,3 +231,17 @@ func TelemetryOptedOut(optoutVar string) bool {
 
 	return optoutVar == "1" || optoutVar == "true"
 }
+
+// IsPluginCommand returns true if the command invoked is for a plugin
+// flase otherwise
+func IsPluginCommand(cmd *cobra.Command) bool {
```

Comment:
nit: Would be be better off outside this package? I can see it coming handy for non-telemetry related reasons in the future

Reply:
this is a great catch and I agree!